src/common/sound.cpp

```diff
@@ -174,12 +174,16 @@ void Sound::_sound(int rep, float frq, int16_t dur, int16_t del, float vol /*, b
         return;
     }
 
-    /// store variables for timing method
+    /// store ACTIVE variables for timing method
     repeat = rep;
     frequency = frq;
     duration_set = dur;
     delay_set = del;
     volume = (vol * varVolume) * 0.3F;
+    /// for debugging BSOD
```

Comment:
#ifdef _DEBUG

Reply:
No need to put it in macro. Debug mode is added in sound_enums.cpp - so only macro that is staying is in MItem_tools.cpp.